python/lsst/analysis/tools/actions/vector/selectors.py

```diff
@@ -76,9 +76,6 @@ def getInputSchema(self) -> KeyedDataSchema:
     def __call__(self, data: KeyedData, **kwargs) -> Vector:
         """Select on the given flags
 
```

Comment:
Does this belong on this ticket? It seems like a selectors fix, but is just removing `Parameters` without replacing it. From what I can tell, this looks like it needs `data` at least?